src/test/java/dal/fatih/todoly/controller/TaskControllerTest.java

```diff
@@ -0,0 +1,523 @@
+package dal.fatih.todoly.controller;

```

Comment:
You can rename the Test to be `TodolyTest`. This is a component test which means that you are supposed to test more than controller